src/components/themes/purple-haze/theme.css

```diff
@@ -24,7 +24,6 @@ html {
 }
 
 .skinHeader.semiTransparent {
-    -webkit-backdrop-filter: none !important;
     backdrop-filter: none !important;
     background-color: rgba(0, 0, 0, .3);
     background: -webkit-gradient(linear, left top, left bottom, from(rgba(0, 0, 0, .6)), to(rgba(0, 0, 0, 0)));
```

Comment:
This should be a gradient right? Might need to run a search for this as well.

Reply:
I generally cleaned up the prefixed gradients. I don't know what the original idea was for adding a background color AND a gradient. Perhaps fallback for an old IE version that was never removed ?

I think we can safely get rid of it.

Reply:
I think it was indeed support for older browsers, so we can get rid of it for now.